.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules
 
 # Generated files
 lib
+dist
```

Comment:
Right now module being exported into `lib` so if you change it here you have to updated `package.json` as well.